test/BUILD

```diff
@@ -777,6 +777,43 @@ scala_junit_test(
     runtime_deps = [":JunitRuntimePlatform"],
 )
 
+scala_library(
+    name = "JunitNoTestEnvironment",
+    srcs = [
+        "src/main/scala/scalarules/test/junit/JunitNoTestEnvironmentTest.scala",
+    ],
+    deps = ["@io_bazel_rules_scala_junit_junit"],
+)
+
+scala_library(
+    name = "JunitSetTestEnvironment",
+    srcs = [
+        "src/main/scala/scalarules/test/junit/JunitSetTestEnvironmentTest.scala",
+    ],
+    deps = ["@io_bazel_rules_scala_junit_junit"],
+)
+
+scala_junit_test(
+    name = "JunitNoTestEnvironment_test_runner",
+    size = "small",
+    runtime_jdk = "@bazel_tools//tools/jdk:remote_jdk11",
+    suffixes = ["Test"],
+    tests_from = [":JunitNoTestEnvironment"],
+    runtime_deps = [":JunitNoTestEnvironment"],
+)
```

Comment:
I wonder if this can be a `scala_junit_test` that directly declares `srcs` on `src/main/scala/scalarules/test/junit/JunitNoTestEnvironmentTest.scala`. ditto for the other, so the BUILD file can be simplified a little.

Reply:
Good idea; don't want to get into it in this commit, but we can track it as an issue?

Reply:
It should be able to do that already - https://github.com/bazelbuild/rules_scala/blob/b85d1225d0ddc9c376963eb0be86d9d546f25a4a/test_expect_failure/scala_junit_test/BUILD#L10-L14, so should be minimal change?

Reply:
Nice catch! Simplified it.